test/test_shared.cpp

```diff
@@ -2082,7 +2082,7 @@ TEST_IF(Shared_AsyncMultiprocess, allow_async)
 
 #if !defined(_WIN32)
 // this test does not work with valgrind:
```

Comment:
Does it work with valgrind now? (It would be great if it does!) Otherwise we will probably get a report about it after this is merged to master.